dice.js

```diff
@@ -0,0 +1,9 @@
+const rollDice = function(){
+    let randomNum1 = 6 - Math.floor(Math.random() * Math.floor(6));
+    let randomNum2 = 6 - Math.floor(Math.random() * Math.floor(6));
```

Comment:
- You don't need to use `Math.floor()` with a **whole number**.  The output of `Math.floor(6)` is `6`, so just type 6
- Not sure why you're subtracting 6 at the beginning?